ajax/libs/webtorrent-component/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "webtorrent-component",
+  "filename": "webtorrent.component.min.js",
+  "description": "WebTorrent HTML element.",
+  "author": "Mikeal Rogers",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mikeal/webtorrent-element"
+  },
+  "keywords": [],
```

Comment:
Could you enrich some keywords for us? Like `html`, `WebTorrent`, `element`? Thanks.